scripts/test/__init__.py

```diff
@@ -0,0 +1,4 @@
+#! /usr/bin/env python3
+
+sys.path.append('/scr/tmp/taylort/aircraft_projects/scripts/')
```

Comment:
I was able to successfully run the tests on my mac (Yay!) but had to change this path. I suggest you add a README.md to this test/ dir that lets users know that to run the tests they will have to 1) change this path to wherever their checkout is, and 2) from the scripts dir (not inside test/) run "python -m unittest discover -s test -v"

Reply:
Thanks Janine! I just committed a README.md within the test subdir that has this information.